docs/v0.5/zh/user-guide/query-data/sql.md

```diff
@@ -111,38 +111,38 @@ SELECT * FROM monitor WHERE host='127.0.0.1' AND ts > '2022-11-03 03:39:57';
 ### 使用时间索引过滤数据
 
 按照时间索引来过滤数据是时序数据库的一个关键特性。
-GreptimeDB 支持 `RFC3339`、`ISO8601` 和 UNIX 时间戳格式来过滤数据，以便你更方便的写时间索引的约束条件。
-
-例如，下面的查询展示了三种格式的时间索引过滤数据的用法：
+默认情况下，数据库会将条件中的时间类型视为该列的值类型。
+例如，假如 `monitor` 表中的 `ts` 列的值类型为 `TimestampMillisecond`，你可以使用下面的查询来过滤数据：
 
 ```sql
--- RFC3339
-SELECT * FROM monitor WHERE ts > '2022-11-03 03:39:57Z';
--- ISO8601
-SELECT * FROM monitor WHERE ts > '2022-11-03T03:39:57+08:00';
--- UNIX timestamp
+-- The unix time value 1667446797000 corresponds to the TimestampMillisecond type.
 SELECT * FROM monitor WHERE ts > 1667446797000;
 ```
 
-你还可以使用时间函数来过滤数据。
-<!-- Here are some examples of using time and date functions in the `WHERE` clause. -->
-例如，使用 `now()` 函数和 `INTERVAL` 关键字来获取最近 5 分钟的数据：
+当过滤条件中的时间值类型与时间索引列的值类型不同时，
+你需要使用 `::` 语法来指定时间值的类型来确保数据库正确的识别时间值的类型。
+例如数字 `1667446797` 表示一个以秒为单位的时间戳，
+你可以使用 `::TimestampSecond` 语法来指定它的类型为 `TimestampSecond`，
```

Comment:
Should we list all available options here? The full set is:

- second: "TIMESTAMP_S" | "TIMESTAMP_SEC" | "TIMESTAMPSECOND"
- millisecond: "TIMESTAMP_MS" | "TIMESTAMPMILLISECOND"
- microsecond: "TIMESTAMP_US" | "TIMESTAMPMICROSECOND"
- nanosecond: "TIMESTAMP_NS" | "TIMESTAMPNANOSECOND"

If this is too verbose, we can only give the first short type

Reply:
I will write these content to the Data-types doc.